fuzz/fixture-fd/src/feature_set.rs

```diff
@@ -1,20 +1,20 @@
 //! Runtime feature set.
 
 use {
-    super::proto::FeatureSet as ProtoFeatureSet, solana_feature_set::FeatureSet,
+    super::proto::FeatureSet as ProtoFeatureSet, agave_feature_set::FeatureSet,
     solana_keccak_hasher::Hasher, solana_pubkey::Pubkey,
 };
 
 // Omit "test features" (they have the same u64 ID).
 pub static OMITTED_FEATURES: &[Pubkey] = &[
-    solana_feature_set::disable_sbpf_v0_execution::id(),
-    solana_feature_set::reenable_sbpf_v0_execution::id(),
+    agave_feature_set::disable_sbpf_v0_execution::id(),
+    agave_feature_set::reenable_sbpf_v0_execution::id(),
 ];
 
 impl From<ProtoFeatureSet> for FeatureSet {
     fn from(value: ProtoFeatureSet) -> Self {
         let mut feature_set = Self::default();
-        let mut inactive = std::mem::take(&mut feature_set.inactive);
+        let mut inactive = feature_set.inactive().clone();
```

Comment:
I am not sure whether this needs the "take" or not – cloning should have the same effect since the `feature_set` has just being created.